beebop/app.py

```diff
@@ -54,36 +53,25 @@ def check_connection(redis):
         abort(500, description="Redis not found")
 
 
-def generate_zip(path_folder):
+def generate_zip(path_folder, type, cluster):
```

Comment:
You might want to refactor the nasty with/for/for/if bit into its own function and test it, but up to you